community/good-member/README.md

```diff
@@ -0,0 +1,133 @@
+In it he quotes Ray Dalio saying "If you don't look back at yourself and think, 'Wow how sutpid I was a year ago', then you must not have learned much in the last year".
+
+## You don't know what you don't know
+
+Exercism is built on the tenant that we don't know what we don't know.
+The whole platform works around mentors pointing out knowledge gaps that you didn't know you have.
+And that works incredibly when both sides are open to knew information (very often its the mentors that realise they didn't know something when seeing new code from a student!)
+
+It's ease to remember that in the context of being a student in a mentoring session, but it's just as important everywhere else within Exercism.
+When you think something sucks about the website, start by asking yourself what it is that you might not know — a decision that looks ridiculous to you might indicate that your knowledge is missing something just as much as it might indicate that the decision actually **is** ridiculous.
+
+Take a read of our post on [Chesteron's Fence](./being-a-good-community-member/chestertons-fence) to practical examples of how this principle applies.
+
+## Disagree constructively
+
+A huge part of building a successful product is in people bringing different opinions and discussing them to aim to find truth (or at the very least some constructive way forward).
+Disagreeing with each other is therefore a critical component of Exercism.
+However, disagreeing constructively is a skill — one that is difficult to master.
+
+Here a few guiding points on how to disagree constructively within our community:
+
+### 1. Ask yourself if you need to get involved
+
+So you've found a problem and feel like someone needs to know about it.
+The first thing to ask yourself is do they?
+Will telling someone that something is wrong make things better?
+Are you adding noise or signal?
+
+More often than not, telling someone that they're wrong is great.
+You help them address a knowledge gap and help make Exercism better.
+But sometimes it's not worth the noise or energy, and correcting someone just makes someone feel worse about themselves.
+So before you jump in, just sanity-check whether it's the right thing to do.
+
+### 2. Don't pile in
+
+Closely linked to 1 is our tribal nature of seeing something we have an opinion on and feeling the need to comment.
+You see this a lot on GitHub with people adding 👍 and 👎 emojis onto every comment or spamming a thread with "Me too".
+Many people see two people disagreeing and feel they need to "reinforce" the person that is saying what they agree with.
+
+If you're not saying something new, or something useful, ask yourself whether you need to say anything at all.
+
+- Does that 👎 just make someone feel rubbish about themselves?
+- Is it going to just entrench them further in their position and make them defensive?
+- Does adding a 👍 make someone else feel like they're right, boost their ego, and entrench them further.
```

Comment:
I think the thumbs up/down emoji are a great way to prevent people from piling on. It clearly shows you support something but it doesn't spam people will notifications, so it's not as overwhelming. In some ways it's the virtual equivalent to nodding your head in a meeting, and I don't think anyone would consider that a "pile on".

Reply:
I find it piling on. It's one of my least favourite things about working in open source when you say something and lots of people :-1: you or when someone you're disagreeing with gets lots of :+1:. IMO it:
- Turns many debates into a popularity contests
- Makes people more defensive and more entrenched
- Creates tribes who agree or disagree en mass with things.

If you look at all the problematic PS discussions you'll see that this was a behaviour that kicked in there.

I do understand the perspective that it's helpful to vote up and down on things, or show agreement, but in the context of a debate, it feels to me like an audience cheering or booing at every comment, and that definitely means I am less constructive in conversations when I'm in one where that's happening.

Reply:
I see. That's really different to how I feel about and react to emote reactions.

> It's one of my least favourite things about working in open source when you say something and lots of people 👎 you or when someone you're disagreeing with gets lots of 👍

To me it's a good thing in both cases<sup>1</sup>. It means that people have read what I said, and care about it in some way. Imo it's worse to only receive a single comment because it feels like nobody cares, even if perhaps a dozen people have seen it but were discouraged from weighing in on it due to discouraging the emote reactions.

Personally I think the overall risk of discouraging them (more notification spam because people _will_ comment, the feel that nobody gives a damn about what one wrote) far outweighs the advantages.

Maybe we could discourage piling on a comment with emotes, but still encourage the reactions generally?

Something along the lines of

> Please consider if it really adds anything to be the fifth person up- or downvoting a comment.

<sup>1</sup>Unless the issue was linked to some other place and it's just randoms voting on it which I don't think happens a lot in Exercism repos and can be stopped through mod tools when it happens.